e2e/playwright/playwright.config.ts

```diff
@@ -30,6 +30,12 @@ export default defineConfig({
 
     /* Screenshot on failure. */
     screenshot: 'only-on-failure',
+
+    /* Timeout for each action in milliseconds. Defaults to 0 (no limit). */
+    actionTimeout: 10 * 1000,
+
+    /* Timeout for each navigation in milliseconds. Defaults to 0 (no limit). */
+    navigationTimeout: 30 * 1000,
```

Comment:
If I understand this change correctly, it will make my dev workflow difficult as I often comment things out and manually navigate to the point at which my test can start.

Additionally, this may affect other tests that are already passing.